.github/workflows/e2e.yml

```
@@ -43,7 +43,7 @@ jobs:
             name: iOS
             appPlain: performance-tests/test-app-plain.ipa
           - platform: android
-            runs-on: ubuntu-latest
+            runs-on: ubuntu-22.04
```

I would add a comment linking to the reason why we did this so in the future we have a reference on what to look for when updating the runner

```suggestion
            # Not using the latest version due to a known issue: https://github.com/getsentry/sentry-react-native/issues/4418
            runs-on: ubuntu-22.04
```

---

Good idea to document this @lucas-zimerman 🙇